Components/Buttons/button.css

```diff
@@ -1,27 +1,33 @@
 .btn {
   background-color: var(--secondary);
-  cursor: pointer;
-  padding: 0.5rem 1rem;
-  font-size: 1.5rem;
+  font-size: 1.3rem;
   border: 1px solid transparent;
   border-radius: 0.475rem;
```

Comment:
Any specific reason for using such a specific value?

Reply:
default font size of button text is too small to read. I felt 1.3rem is a sweet spot and has good readbility. So, hardcoded the value.